src/src/Helpers/Networking.cpp

```diff
@@ -1685,11 +1685,11 @@ int http_authenticate(const String& logIdentifier,
 
     # if FEATURE_OMETEO_EVENT

```

Comment:
IMHO, a somewhat better name for this would be `FEATURE_OPENMETEO_EVENT`, ometeo is a bit ambiguous, I think.

Also, this variable, and a small summary of the description, should go in the `Custom-sample.h` file, and a check with default into `define_plugin_sets.h`:
```
#ifndef FEATURE_OPENMETEO_EVENT
  #define FEATURE_OPENMETEO_EVENT 0 // Or maybe 1 for ESP32 (MAX) builds
#endif
```